### PR TITLE
scripted-diff: Modernize nLocalServices naming

### PR DESCRIPTION
**Pull request by fjahr**
The type of the `nLocalServices` variable was changed to `std::atomic<ServiceFlags>` in #30807 and I suggested the variable name to get updated with a scripted diff along with it. It wasn't included in the PR but I am still suggesting to do it as a follow-up since I had already prepared the commit.

### CODE REVIEW

File: src/init.cpp [hunk 1]
```diff
@@ -840,7 +840,7 @@ namespace { // Variables internal to initialization process only
 
 int nMaxConnections;
 int available_fds;
-ServiceFlags nLocalServices = ServiceFlags(NODE_NETWORK_LIMITED | NODE_WITNESS);
+ServiceFlags local_services = ServiceFlags(NODE_NETWORK_LIMITED | NODE_WITNESS);
```
[the remark this answers is not in the document]
Comment (fjahr) [reply src/init.cpp]: Hm, ok, I interpreted the comment on the namespace above as that we ignore that here. But I missed the variable below and I the others are just older. Changed.

**Review by furszy — COMMENTED** [review]
utACK 33381ea530ad79ac1e04c37f5707e93d3e0509ca

**Review by jonatack — COMMENTED** [review]
ACK 33381ea530ad79ac1e04c37f5707e93d3e0509ca

Following the last push, suggest renaming the pull title to `scripted-diff: Modernize nLocalServices naming` (edit: and also the commit name)

**Comment by fjahr** [comment]
> Following the last push, suggest renaming the pull title to `scripted-diff: Modernize nLocalServices naming` (edit: and also the commit name)

done

**Comment by mzumsande** [comment]
For the record, I don't like refactor PRs that just rename things to a new convention. It's fine for me if the code is touched anyway, but opening a PR just for that? We probably have hundreds of variables that could be modernized the same way.

**Comment by fjahr** [comment]
> For the record, I don't like refactor PRs that just rename things to a new convention. It's fine for me if the code is touched anyway, but opening a PR just for that? We probably have hundreds of variables that could be modernized the same way.

Sure, I agree and I had not opened it if I had not already written the PR and had been encouraged to open it as a follow-up even though the PR where the code was touched was merged without it, see https://github.com/bitcoin/bitcoin/pull/30807#issuecomment-2338236543.

**Review by theStack — APPROVED** [review]
ACK 33381ea530ad79ac1e04c37f5707e93d3e0509ca

**Comment by Javadseyedi12** [comment]
Hello, good time, yes, what should I do?

در تاریخ دوشنبه ۳۰ سپتامبر ۲۰۲۴،‏ ۱۸:۱۵ Pieter Wuille <
***@***.***> نوشت:

**Comment by achow101** [comment]
ACK 33381ea530ad79ac1e04c37f5707e93d3e0509ca

**Pull request merged**